frontend/packages/dev-console/src/plugin.tsx

```diff
@@ -492,6 +512,18 @@ const plugin: Plugin<ConsumedExtensions> = [
         )).default,
     },
   },
+  {
+    type: 'Page/Route',
+    properties: {
+      perspective: 'dev',
+      exact: false,
```

Comment:
Exact should be ```true```.

Reply:
Having exact as `true`, Tab routes doesn't work & we get 404. Need to check with @christianvogt regarding this.

Reply:
I had similar issues with Project Details page. Christian and I opted out of using `false` as it can lead to unintentional matches. Since this is a new route, I am unsure it'll be of a big deal either way.

If you want an example of what I did, see:
```
  {
    type: 'Page/Route',
    properties: {
      perspective: 'dev',
      exact: true,
      path: ['/k8s/cluster/projects'],
      loader: async () =>
        (await import(
          './components/projects/details/AllProjectsDetailList' /* webpackChunkName: "all-projects-detail-list" */
        )).default,
    },
  },
  {
    type: 'Page/Route',
    properties: {
      perspective: 'dev',
      path: ['/k8s/cluster/projects/:name'],
      loader: async () =>
        (await import(
          './components/projects/details/ProjectDetailsPage' /* webpackChunkName: "project-details" */
        )).default,
    },
  },
```
In this file.

Reply:
@andrewballantyne  actually `exact` defaults to `false` so now I've split the routes & followed Project Details page approach.

Reply:
Ah, right you are... it was late when I made that comment :)